services/core-api/app/api/now_applications/resources/now_application_export_resource.py

```diff
@@ -320,7 +388,20 @@ def transform_documents(now_application):
                     doc['now_application_document_type_code'])
                 doc['now_application_document_type_description'] = get_description(
                     document_type, doc['now_application_document_type_code'])
-                included_docs.append(doc)
+                if doc['is_final_package']:
```

Comment:
Are there any potential side effects to other docs without this key?

Reply:
Not that I could see, all Docs will have this key, but it may be null or false, in those cases its ignored.